templates/view.erb

```diff
@@ -34,6 +34,15 @@ view "<%= @name %>" {
 	};
 <%-   end -%>
 <%- end -%>
+<%- if @server -%>
+<%-   @server.each_pair do |srv, srv_values| -%>
+	server <%= srv %> {
+<%-		Array(srv_values).each do |srv_value| -%>
+		<%= srv_value %>;
```

Comment:
This is what I was referring to. In this loop, each element of `srv_values` becomes a complete line of configuration as-is. It gets the job done, but it also permits arbitrary text in the templated configuration file. I think that the template should respond to specific keys in the server hash.